tests/src/org.mockito/mockito-core/4.8.1/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+org.mockito.internal.creation.proxy.ProxyMockMaker
```

Comment:
What this file represents? Please explain its purpose

Reply:
It switches the MockMaker from Mockito to the ProxyMockMaker, which uses JDK proxies to create mocks.

Reply:
See here https://site.mockito.org/javadoc/current/org/mockito/plugins/MockMaker.html